_maps/map_files/Delta/delta.dmm

```diff
@@ -70720,6 +70735,21 @@
 	},
 /turf/simulated/floor/plasteel/white,
 /area/station/medical/medbay)
+"gfR" = (
+/obj/structure/cable{
+	d2 = 8;
+	icon_state = "0-4"
+	},
+/obj/machinery/power/apc{
+	dir = 8;
```

Comment:
Same thing Warriorstar said over your meta PR, no need for the dir varedit.

Reply:
fixed